build.ps1

```diff
@@ -1,19 +1,16 @@
 $ErrorActionPreference = "Stop"
 
 $cibuild = "false"
-
-# Make sure that we have something on non-bots
-if (!$env:BUILD_NUMBER) {
-    $env:BUILD_NUMBER = "0"
-}
```

Comment:
you don't actually need to remove the lines above, right? the less changes we have compared to upstream, the better

Reply:
I can restore it, but note that this variable is no longer used in the script.